docs/guides/admin/sso.md

```diff
@@ -27,15 +29,22 @@ Give your application a **meaningful** *Name* and a *Description*.
 
 Next, set the connection type to *Authentication Only* and specify the *Callback URL*
 
-**Note on the Callback URL**: This is the URL that the user should be redirected to once authentication was successful. In the example, we can see it is `https://seat.test/auth/eve/callback`. You should replace the `seat.test` part with *your* domain! In other words, lets say you are hosting SeAT at `https://this.is.seat/`, then the Callback URL will be `https://this.is.seat/auth/eve/callback`. If you have SeAT in a sub folder on your web server, remember to prepend the folder name before `/auth/eve/callback`.
+!!! note "Note on the Callback URL"
+
+    This is the URL that the user should be redirected to once authentication was successful.
+    In the example, we can see it is `https://seat.test/auth/eve/callback`.
+    You should replace the `seat.test` part with *your* domain!
+    In other words, lets say you are hosting SeAT at `https://this.is.seat/`, then the Callback URL will be
```

Comment:
i would not be so specific. Maybe put the seat.test in curly parenthesis:

`https://{your.domain}/auth/eve/callback`